### DISCUSSION TITLE
hwo to enable aws-xray with fast api?

### DISCUSSION THREAD

**Discussion by ca8888** [fastapi/fastapi#3505] [Questions] [answered]
I am trying to understand how can I enable xray for fast api
from aws documentation i only found django examples!
would be thankful if someone can help me figure it out

**Comment by ghandic** [comment, marked as answer]
This is a request for xray, however, it looks like there is an issue open for it. Maybe ask on there? https://github.com/aws/aws-xray-sdk-python/issues/270
